lib/src/core/provider/calendar_scroll_provider.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:developer';
+
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+class CalendarScrollProvider with ChangeNotifier {
+
+  CalendarScrollProvider(){
+    var currentMonth = DateTime.now();
+    var formatter = DateFormat('LLLL');
+    selectedMonth =  formatter.format(currentMonth);
+    indexedDates = getIndexedDates ();
+  }
+
+  FixedExtentScrollController fixedExtentScrollController;
+  int currentItem = 20;
+  int numberAmount = 61;
+  int indexStartsFrom = 0;
+  String selectedMonth;
+  List<Map<String,  dynamic>> indexedDates;
```

Comment:
do we need to store all data about the current date except a day that is used in the scroll?